src/pgmigrations/1693317600000_create_worlds_table.ts

```diff
@@ -0,0 +1,21 @@
+import { MigrationBuilder, PgType } from 'node-pg-migrate'
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  pgm.createTable('worlds', {
+    name: { type: PgType.VARCHAR, notNull: true, primaryKey: true },
+    owner: { type: PgType.VARCHAR, notNull: false },
```

Comment:
Can a world be created without owner?

Reply:
Actually... we don't store the owner, we store the wallet that deployed it. It could be the owner, or it could be an authorized wallet.
In the future, we might want to populate some owner column. For now, I'm just going to remove the column.